src/neuroconv/datainterfaces/ecephys/openephys/openephysanaloginterface.py

```diff
@@ -0,0 +1,203 @@
+            )
+        if stream_name is not None and stream_name not in available_streams:
+            raise ValueError(
+                f"The selected stream '{stream_name}' is not in the available streams '{available_streams}'!"
+            )
+
+        self.stream_name = stream_name or available_streams[0]
+        self.block_index = block_index
+
+        # Initialize the recording extractor
+        self.recording_extractor = OpenEphysBinaryRecordingExtractor(
+            folder_path=folder_path,
+            stream_name=self.stream_name,
+            block_index=block_index,
+        )
+
+        # Filter for only analog channels (ADC)
+        channel_ids = self.recording_extractor.get_channel_ids()
+        self.analog_channel_ids = [id for id in channel_ids if "ADC" in id]
+
+        if not self.analog_channel_ids:
+            raise ValueError(f"No analog channels (ADC) found in the selected stream '{self.stream_name}'!")
+
+        # Select only analog channels
+        self.recording_extractor = self.recording_extractor.select_channels(channel_ids=self.analog_channel_ids)
+
+        super().__init__(
+            folder_path=folder_path,
+            stream_name=stream_name,
+            block_index=block_index,
+            verbose=verbose,
+            time_series_key=time_series_key,
+        )
+
+    def get_metadata(self) -> dict:
+        metadata = super().get_metadata()
+
+        session_start_time = _get_session_start_time(element=self._xml_root)
+        if session_start_time is not None:
+            metadata["NWBFile"].update(session_start_time=session_start_time)
+
+        return metadata
+
+    def get_channel_names(self) -> list[str]:
+        """
+        Get a list of analog channel names from the recording extractor.
+
+        Returns
+        -------
+        list of str
+            The names of all analog channels in the OpenEphys recording.
+        """
+        return list(self.analog_channel_ids)
+
+    def add_to_nwbfile(
+        self,
+        nwbfile: NWBFile,
+        metadata: Optional[dict] = None,
+        stub_test: bool = False,
+        iterator_type: Optional[str] = "v2",
+        iterator_opts: Optional[dict] = None,
+        always_write_timestamps: bool = False,
+    ):
+        """
+        Add analog channel data to an NWB file.
+
+        Parameters
+        ----------
+        nwbfile : NWBFile
+            The NWB file to which the analog data will be added
+        metadata : Optional[dict], default: None
+            Metadata dictionary with device information. If None, uses default metadata
+        stub_test : bool, default: False
+            If True, only writes a small amount of data for testing
+        iterator_type : Optional[str], default: "v2"
+            Type of iterator to use for data streaming
+        iterator_opts : Optional[dict], default: None
+            Additional options for the iterator
+        always_write_timestamps : bool, default: False
```

Comment:
I think I'd use "force" instead of "always" but it's not a big deal

Reply:
It is named like this in other interfaces. I agree force would be better. Should we change it globally?